src/supermin-init-prelude.sh

```diff
@@ -25,11 +28,19 @@ fi
 mkdir -p "${workdir}"/cache /host/container-work
 mount /dev/sdb1 "${workdir}"/cache
 
+if [ -f "${workdir}/tmp/supermin/supermin.env" ]; then
+    source "${workdir}/tmp/supermin/supermin.env";
+fi
+
 # /usr/sbin/ip{,6}tables is installed as a symlink to /etc/alternatives/ip{,6}tables but
 # the /etc/alternatives symlink to /usr/sbin/ip{,6}tables-legacy is missing.  This recreates
 # the missing link.  Hehe.
 update-alternatives --install /etc/alternatives/iptables iptables /usr/sbin/iptables-legacy 1
 update-alternatives --install /etc/alternatives/ip6tables ip6tables /usr/sbin/ip6tables-legacy 1
 
+# some commands run chroot, which is not needed when you are root
+sudo() { \${@}; }
+export -f sudo
```

Comment:
not sure I exactly understand this - what commands are running `chroot` ?

Reply:
`coreos-assembler` itself.

Reply:
link to example?

Reply:
I'm actually mistaken on the chroot front...that was removed. Well done challenging that 👍

Reply:
https://github.com/coreos/coreos-assembler/blob/master/src/coreos-assembler#L27 was the code path that I was worried about, and https://github.com/coreos/coreos-assembler/blob/master/src/cmdlib.sh#L131 calls sudo.

Reply:
i could understand if it was `sudo` and not `chroot`

If that were the case I would update the comment to: `some commands run via sudo, but that's not needed inside supermin as we are running as root and sudoers isn't set up, override sudo with a function to override this`.

Reply:
Removed, and pushed.